cli/cmd/check.go

```diff
@@ -103,14 +107,21 @@ func configureAndRunChecks(options *checkOptions) {
 
 	success := runChecks(os.Stdout, hc)
 
-	fmt.Println("")
```

Comment:
This `Println` separates the final result from the checks above it.

```
kubernetes-api: can initialize the client..................................[ok]
kubernetes-api: can query the Kubernetes API...............................[ok]
...
linkerd-version: control plane is up-to-date...............................[ok]

Status check results are [ok]
```

This would have been caught by our integration tests, but unfortunately those do not run on every PR, only master merge.

To run integration tests (make sure you have access to a kubernetes cluster, and can create:
```bash
bin/docker-build-cli-bin
bin/test-run `pwd`/bin/linkerd checktest
```

... or to run a relevant subset:
```bash
bin/docker-build-cli-bin
go test test/install_test.go --linkerd `pwd`/target/cli/darwin/linkerd --integration-tests --linkerd-namespace checktest
```

and then clean up:
```bash
bin/test-cleanup checktest
```

Reply:
LOL! I'll push it back with a comment then :D That totally looked like something left my mistake